src/nepiada/utils/graphs.py

```diff
@@ -7,38 +7,53 @@
 
 class Graph:
     def __init__(self, config, agents, screen=None, cell_size=0):
-        print("Graphs have been initialized")
-
         self.dim = config.size
         self.agents = agents
         self.dynamic_obs = config.dynamic_obs
-        self.full_communication = config.full_communication
+        self.dynamic_comms = config.dynamic_comms
```

Comment:
I like the naming convention here, good descriptive names

Reply:
Thanks!

Reply:
Thanks!